modules/deploying-models-on-the-single-model-serving-platform.adoc

```diff
@@ -35,7 +35,9 @@ endif::[]
 ifdef::upstream[]
 * To use the vLLM runtime or use graphics processing units (GPUs) with your model server, you have enabled GPU support. This includes installing the Node Feature Discovery and GPU Operators. For more information, see https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on {org-name} OpenShift Container Platform^] in the NVIDIA documentation.
 endif::[]
-
+*  To deploy {rhelai-productname-short} models:
+** You have enabled the vLLM runtime.
+** You have downloaded the model from the registry and uploaded it to S3-compatible object storage.
```

Comment:
Few questions here:

- Can we deploy _rhelai_ models with other runtimes? Or is it only possible to deploy them with the vLLM runtime? We might need a little more clarification here.
- Which registry? Suggestion: Add a link to the supported _rhelai_ models list/model registry.

Reply:
@tarukumar Any thoughts here?

Reply:
Edits from discussion with Shumaila and Tarun
```suggestion
** You have downloaded the model from the {org-name} container registry and uploaded it to S3-compatible object storage.
```